rfcs/text/pe/pe.yml

```
@@ -0,0 +1,168 @@
+---
+- name: pe
+  type: group
+  fields:
+    - name: main_icon
+      level: extended
+      type: object
+      description: >
+        Hashes of embedded program icon.
+      fields:
+        - name: dhash
+          level: extended
+          type: keyword
+          description: >
+            Difference Hash (dhash) for a given PE file.
+
+        - name: md5
+          level: extended
+          type: keyword
+          description: >
+            MD5 hash of raw icon data.
+
+    - name: debug
+      level: extended
+      type: keyword
+      description: Debug information, if present
+
+    - name: import_list
+      level: extended
+      type: keyword
+      description: List of all imported functions
+
+    - name: sections
+      level: extended
+      description: >
+        Data about sections of compiled binary PE
+      type: keyword
+
+    - name: resource_details
+      level: extended
+      type: keyword
+      description: >
+        If the PE contains resources, some info about them
+
+    - name: resource_languages
+      level: extended
+      type: keyword
+      short: List of resource languages.
+      description: >
+        Digest of languages found in resources. Key is language (as string) and
+        value is how many resources there are having that language (as integer)
+
+    - name: resource_types
+      level: extended
+      type: keyword
+      short: List of resource types.
+      description: >
+        Digest of resource types. Key is resource type (as string) and value
+        is how many resources there are of that specific type (as integer)
+
+    - name: packers
+      level: extended
+      type: keyword
+      short: Identifies packers used.
+      description: >
+        Identifies packers used on Windows PE files by several tools and AVs.
+        Keys are tool names and values are identified packers, both strings.
+        see `file.pe.packers` for merged list of packers from all tools.
+
+    - name: exports
+      level: extended
+      type: keyword
+      description: >
+        List of symbols exported by PE
+
+    - name: creation_date
+      level: extended
+      short: Build or compile date.
+      description: >
+        Extracted when possible from the file's metadata. Indicates when it was
+        built or compiled. It can also be faked by malware creators.
+      type: date
```

When adding examples here, it's as simple as adding another attribute called `example:`.

```suggestion
      type: date
      example: "2020-11-05T17:25:47.000Z"
```

If the example is a composite value like an array or a JSON blob, make sure you quote it.

If the example is a string, quoting is optional.

If the example is a float or int, don't quote it.